src/layouts/sidebar-layout/sidebar-layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
```

Comment:
Oh, that is dope. So basic all the routes render inside of this wrapper, right?

Reply:
yup, all child `Route`(s) of this layout